Button/button.css

```diff
@@ -0,0 +1,189 @@
+:root{
+    --primary-color:rgb(61, 205, 224);
+    --secondary-color:rgb(133, 173, 233);
+    --red-color:rgb(155, 8, 8);
+    --white-color:white;
+    --btn-padding:0.8rem 1.7rem 0.8rem 1.7rem ;
+    --btn-size:1.2rem;
+    --btn-weight:bold;
+    --btn-border:0;
+}
+button{
+    cursor: pointer;
+    color: white;
+}
+.code{
+    margin-top: 2rem;
+}
+.header{
+    display: flex;
+    justify-content: space-between;
+    background-color:rgb(139, 185, 179);
+    color:black;
+    font-weight: var(--btn-weight);
+    font-size:var(--btn-size);
+    padding:1rem;
+    border-radius: 1rem;
+    position: sticky;
+    top: 0;
+    z-index: 1;
+}
+.home{
+    display: flex;
+    gap:0.5rem;
+}
+.home-link{
+    margin-top:1rem;
+    text-decoration: none;
+    color:black;
+}
+.container{
+    width:100%;
+    display: flex;
+    gap: 7rem;
+}
+.sidebar-container{
+    display: flex;
+    flex-direction: column;
+    gap: 3rem;
```

Comment:
Use global variables for properties like margin, padding, and gap to maintain consistency across the website, something like --space-xsmall: 0.5rem

Reply:
ok will do that